### ISSUE TITLE
Where `.timetrap-sheet` does not honor child folders location for `t out`

### ISSUE THREAD

**Issue by kotp** [samg/timetrap#109]
If I am in a child folder of a project and I want to check out of timetrap it should honor the same sheet.  I have several sheets and noticed this today.

I don't often change into sub-folders, but it surprised me, as it should probably search up the folder chain and look for that file.

What happened for me is that it complained that I was not on the "other project" time sheet when I did `t out`.

To simplify this:

When I am in a project
and I `t in`
I should start time on that project
when I change to a child folder of that project
and then I `t out`
I should stop time on that project

**Comment by samg** [comment]
I agree that having timetrap look up the directory structure for a .timetrap-sheet file wouild be an improvement.  I believe @UnderpantsGnome did the implementation of the .timetrap-sheet autosheets functionality and might be interested in making that change.  You could also check out the docs on AutoSheets (https://github.com/samg/timetrap#autosheets) since that system was designed to be extensible to match various user workflows.  I won't have time to work on this in the near future but would happily accept a pull request if you or someone else felt like making this improvement.

Thanks!
